ui/css/perf.css

```diff
@@ -49,6 +49,11 @@ to { opacity: 0; }
   color: #999
 }
 
+.graph-legend.series-inactive {
```

Comment:
I didn't notice this before, but you should seperate out the .series-inactive class:

i.e.:
    .graph-legend .series-inactive {
     }

I'll fix this myself, but letting you know for next time. :)